models/transfers/bnb/bep20/transfers_bnb_bep20.sql

```diff
@@ -1,4 +1,5 @@
 {{ config(materialized='view', alias = alias('bep20'),
+        tags = ['dunesql'],
```

Comment:
if we go down this path, we should for sure enhance the whole model

Reply:
what does enhance mean here?